course/templates/course/course_base.html

```diff
@@ -202,6 +202,19 @@ <h4 class="modal-title" id="unenroll-modal-title">
 	</div>
 </div>
 {% endcomment %}
+
+<script>
+$(function() {
```

Comment:
This could have been added to a separate JavaScript file instead of an inline script. Separate JS files are generally easier to maintain. But anyway, I will merge this. It is a small script connected to the course pages like this html template.